sentry-rails/spec/sentry/rails/action_cable_spec.rb

```diff
@@ -0,0 +1,85 @@
+if defined?(ActionCable) && ActionCable.version >= Gem::Version.new('6.0.0')
+  require "spec_helper"
+  require "action_cable/engine"
+  require "action_cable/connection/test_case"
+  require "action_cable/channel/test_case"
+
+  # ensure we can access `connection.env` in tests like we can in production
+  ActiveSupport.on_load :action_cable_channel_test_case do
+    class ::ActionCable::Channel::ConnectionStub
+      def env
+        @_env ||= ::ActionCable::Connection::TestRequest.create.env
+      end
+    end
+  end
+
+  class ChatChannel < ::ActionCable::Channel::Base
+    def subscribed
+      raise "foo"
+    end
+  end
+
+  class AppearanceChannel < ::ActionCable::Channel::Base
+    def appear
+      raise "foo"
+    end
+  end
+
+  RSpec.describe "Sentry::Rails::ActionCable" do
+    let(:transport) { Sentry.get_current_client.transport }
+
+    before(:all) do
+      make_basic_app
+      ::ActionCable.server.config.cable = { "adapter" => "test" }
+    end
+
+    describe ChatChannel do
+      include ActionCable::Channel::TestCase::Behavior
```

Comment:
can we use [channel spec](https://relishapp.com/palkan/action-cable-testing/docs/channel-specs/channel-spec) here instead of mixing with minitest?